src/ui/mapper.ui

```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>403</width>
-    <height>387</height>
+    <width>613</width>
```

Comment:
This is quite a bit wider - and depending on the size constraints could be problematic for some users - see if you can get by without changing these values...

Reply:
Meh, I didn't change this, that was the qt creator all by itself. in fact I did compensate by shortening the texts. will take another look.

Reply:
Depending on what Git software you use you could have prevented it going into the commit and then reset that unwanted change back out of the codebase...